languages.json

```diff
@@ -26,5 +26,6 @@
 "sv": "swedish",
 "tr": "turkish",
 "uk": "ukrainian",
-"vi": "vietnamese"
+"vi": "vietnamese",
+"fa": "persian/farsi"
```

Comment:
I guess some of the libraries using this repo lookup the value of the json to pick up the file.

Probably would be better to have it either as "farsi" or "persian" only.

Reply:
thanks for your reply, I changed it to "persian"